src/storyblok/Hero.astro

```diff
@@ -1,19 +1,47 @@
 ---
 import { storyblokEditable } from "@storyblok/astro";
+import { Image } from "astro:assets";
 import StoryblokComponent from "@storyblok/astro/StoryblokComponent.astro";
+import type { ButtonStoryblok } from "../../storyblok-component-types";
 
 const { blok } = Astro.props;
-console.log(blok);
 ---
 
-<div {...storyblokEditable(blok)}>
-  <h1>
-    {blok.headline}
-  </h1>
-  <!-- TODO: add rest of field-types -->
-  {
-    blok.button?.map((blok) => {
-      return <StoryblokComponent blok={blok} />;
-    })
-  }
+<div {...storyblokEditable(blok)} class="flex justify-between align-center">
+  <div class="w-1/2">
+    <h1 class="text-7xl font-bold py-10">
+      {blok.headline}<span class="text-orange-500">.</span>
+    </h1>
+    <p>
+      {blok.description}
+    </p>
+    {
+      blok.button?.map((blok: ButtonStoryblok) => {
+        return <StoryblokComponent blok={blok} />;
+      })
+    }
+  </div>
+  <div class="grid grid-cols-5 grid-rows-5">
+    <Image
+      class="col-start-4 row-span-2 col-span-2 rounded-full"
+      src={blok.image_alba.filename}
```

Comment:
Suggestion: I'm wondering if we can have just one "multiple images" field to avoid custom name fields, what do you think? cc @josefineschaefer @schabibi1

I can always take care of that if you think it is worth it, otherwise it's okay this way, just a small improvement

Reply:
@Dawntraoz That could be the solution @josefineschaefer needed✨
This would be much cleaner and less repetitive!

Alba, if you have time, could you do that? I would love to but I'll be on PTO from tmr for a week and I don't want to forget doing it🙏